src/lib/parsers/Args.ts

```diff
@@ -547,8 +549,8 @@ export class Args {
 	 */
 	public next<T>(cb: ArgsNextCallback<T>): T;
 	public next<T>(cb?: ArgsNextCallback<T>): T | string | null {
-		const value = cb ? this.nextMaybe(cb) : this.nextMaybe();
-		return isSome<T | string>(value) ? value.value : null;
+		const value: Option<T | string | null> = cb ? this.nextMaybe(cb) : this.nextMaybe();
```

Comment:
Wait, why is there a typecast? If necessary, you can do `nextMaybe<T>`

Reply:
The typecast is there because `unwrapOr` expects the value to be the same type as the Option. The Option is `T | string` and `null` doesn't fit that:
```
Argument of type 'null' is not assignable to parameter of type 'never'.
```
`unwrapOr` should probably just allow for the value to be of a different type. Regardless, I've done what you've suggested.